docs/user-guide/install-k8s.md

```diff
@@ -1,19 +1,28 @@
-# Sysbox User Guide: Installation in Kubernetes Clusters
+# Sysbox User Guide: Installation in Kubernetes Clusters \[ v0.4.0+ ]
 
-This document describes how to install Sysbox on a Kubernetes cluster (i.e., to
-deploy pods with Sysbox).
+Starting with Sysbox v0.4.0, it's possible to install Sysbox in Kubernetes
+clusters (i.e., to deploy pods with Sysbox).
 
-If you are installing Sysbox on a regular host (i.e., not a Kubernetes host),
-follow [these instructions](install-package.md) instead.
+Such pods are strongly isolated from the host and can seamlessly run
+micro-services or system-level workloads such as systemd, Docker, and even
+Kubernetes in them (much like VMs).
+
+This document describes how to install Sysbox on a Kubernetes cluster. If you
+are installing Sysbox on a regular host (i.e., not a Kubernetes host), follow
+[these instructions](install-package.md) instead.
 
 ## Contents
 
 -   [Kubernetes Version Requirements](#kubernetes-version-requirements)
 -   [Kubernetes Worker Node Requirements](#kubernetes-worker-node-requirements)
 -   [Installation](#installation)
```

Comment:
For consistency purposes i would replace "Installation" with "Installation of Sysbox" to clearly differentiate it from the next section: "Installation of Sysbox-EE".

Reply:
Good idea, done.